packages/config/config/devices/0x0063/ge_12730_zw4002.json

```diff
@@ -15,32 +15,23 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"paramInformation": {
-		"3": {
-			"label": "LED Light",
-			"description": "Sets when the LED on the switch is lit.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "LED on when light off",
-					"value": 0
-				},
-				{
-					"label": "LED on when light on",
-					"value": 1
-				},
-				{
-					"label": "LED always off",
-					"value": 2
-				}
```

Comment:
Parameter 3 is valid, it's only value 2 that does not work. 

```
2021-03-12 20:29:07.588 INFO ZWAVE: Calling api writeValue with args: [ { nodeId: 21, commandClass: 112, property: 3 }, 2, [length]: 2 ]
2021-03-12 20:29:07.589 INFO ZWAVE: Writing 2 to 21-112-0-3
2021-03-12 20:29:07.661 INFO ZWAVE: Success zwave api call writeValue { success: true, message: 'Success zwave api call', result: undefined }
2021-03-12 20:29:08.771 INFO ZWAVE: Node 21: value updated: 112-0-3 2 => 1
```

Reply:
Can you test it? It isn't in the manual for the zw4002 so it would be undocumented. That's possible.

Reply:
Yes, 0 and 1 work fine. I use them in an automation so I can sleep at night. 😄 I wish I could permanently turn it off...
```
2021-03-12 20:39:38.190 DEBUG SOCKET: Event ZWAVE_API emitted to fxJmwvmEqXR2QrSgAAAJ
2021-03-12 20:39:38.192 INFO ZWAVE: Calling api writeValue with args: [ { nodeId: 21, commandClass: 112, property: 3 }, 0, [length]: 2 ]
2021-03-12 20:39:38.192 INFO ZWAVE: Writing 0 to 21-112-0-3
2021-03-12 20:39:38.259 INFO ZWAVE: Success zwave api call writeValue { success: true, message: 'Success zwave api call', result: undefined }
2021-03-12 20:39:39.372 INFO ZWAVE: Node 21: value updated: 112-0-3 0 => 0
2021-03-12 20:39:46.200 DEBUG SOCKET: Event ZWAVE_API emitted to fxJmwvmEqXR2QrSgAAAJ
2021-03-12 20:39:46.203 INFO ZWAVE: Calling api writeValue with args: [ { nodeId: 21, commandClass: 112, property: 3 }, 1, [length]: 2 ]
2021-03-12 20:39:46.204 INFO ZWAVE: Writing 1 to 21-112-0-3
2021-03-12 20:39:46.278 INFO ZWAVE: Success zwave api call writeValue { success: true, message: 'Success zwave api call', result: undefined }
2021-03-12 20:39:47.397 INFO ZWAVE: Node 21: value updated: 112-0-3 1 => 1
```

Reply:
Pushed back in. Online it looks like people paint it with black nail polish.